src/OpenApi/CarterOpenApi.cs

```diff
@@ -170,11 +173,14 @@ private static void CreateOpenApiResponseBody(OpenApiDocument document, KeyValue
                         Type responseType;
                         var responseTypeName = string.Empty;
                         var singularTypeName = string.Empty;
+                        object singleExample = null;
 
                         if (valueStatusCode.Response.IsArray() || valueStatusCode.Response.IsCollection() || valueStatusCode.Response.IsEnumerable())
                         {
                             arrayType = true;
                             responseType = valueStatusCode.Response.GetElementType();
+                            singleExample = ((IEnumerable<object>)valueStatusCode.ResponseExample)?.FirstOrDefault();
```

Comment:
Shouldn't this be set if the response is not an array?

Reply:
Hmm you're right it's weird. I think I put this here because on L201 the existing code constructs a schema for a single item even in the array case, but looking now that never actually gets used in  the case of an array. I think some rearranging to move that inside the if at the bottom can make this go away.

Reply:
Fixed in latest commit